src/battle.ts

```diff
@@ -44,7 +47,7 @@ export interface FaintLogEntry {
 }
 
 interface TurnCommands {
-    [key: number]: TurnCommand | null
+  [key: number]: TurnCommand | null
 }
 
```

Comment:
NIT:
Could be replaced with

```suggestion
type TurnCommands = Record<number, TurnCommand | null>;
```

Reply:
out of scope, refactor of code that does not pertain to MEs